documentation/src/main/asciidoc/topics/code_examples/ConfigRocksDbProperties.java

```diff
@@ -4,7 +4,7 @@
 
 Configuration cacheConfig = new ConfigurationBuilder().persistence()
 				.addStore(RocksDBStoreConfigurationBuilder.class)
-				.location("/tmp/rocksdb/data")
-				.expiredLocation("/tmp/rocksdb/expired")
+				.location("/rocksdb/data")
+				.expiredLocation("/rocksdb/expired")
```

Comment:
To avoid any doubt. This will only work if the global `persistent-location` also starts with `/rocksdb` ... if you want the store to be created under `<persistent-location>/rocksdb/data` etc, you need to remove the `/` at the start of the location as that make it an absolute path.